### ISSUE TITLE
todo: Function for RF resonators

### ISSUE THREAD

**Issue by feschmidt** [feschmidt/pythonGDS#7]
Write a function such as
```python
def resonator (startpoint, endpoint, numberofmeanders, dielectricwidth, conductor width):
    ...
    return resonator
```
 
where
- startpoint: where to start resonator
- endpoint: where to end resonator
- numberofmeanders: the number of meanders
- dielectricwidth : width of dielectric
- conductor width: width of conductor
 
This functions then draws the a cpw resonator with the number of meanders from the start to the end.

Connected to issue [`#14`](https://github.com/feschmidt/stcad/issues/14)

**Comment by feschmidt** [comment]
Also extend RF architecture to gap and finger capacitors and inductive coupling

**Comment by feschmidt** [comment]
Concerning DC bias cavities: fix second launcher. For length adjustments make meandering longer or shorter, and center it. For now this is sufficient.

**Comment by feschmidt** [comment]
General thought: How should we definer the resonators?
- ~~a) Write centerline and gaps on top of each other and do `P-XOR` later on~~
- b) Directly write gaps

**Comment by feschmidt** [comment]
Fixed in [`960ff63`](https://github.com/feschmidt/stcad/commit/960ff63e4b6279c08fe15a68557623ca959d7b42)
